misc/roundtrip

```diff
@@ -40,16 +38,23 @@ if ! [ -x "${UNITS}" ]; then
 fi
 
 
+# disable path expansion, because we don't need it and it's applied on the
+# result of the commands used to loop on, and we don't want that.
+set -f
+# remove space from IFS as it's valid in tag names
+OLD_IFS="$IFS"
+IFS='
+'
+
 for tags in $(find "$UNITS" -name expected.tags); do
     for t in $(sed -e 's/^\([^\t]*\)\t.*/\1/' "$tags"); do
```

Comment:
Use of `\t` breaks OSX support, see https://github.com/universal-ctags/ctags/pull/1605/commits/378c53e653d0db08310a8422a3ed7c2243ed9614

Reply:
Thank you. I merged 378c53e partially.